Source/JavaScriptCore/API/JSScript.h

```diff
@@ -84,22 +84,22 @@ JSC_CLASS_AVAILABLE(macos(10.15), ios(13.0))
 - (BOOL)cacheBytecodeWithError:(out NSError * _Nullable * _Nullable)error;
 
 /*!
- @method
+ @property
  @abstract Returns true when evaluating this JSScript will use the bytecode cache. Returns false otherwise.
  */
-- (BOOL)isUsingBytecodeCache;
+@property (nonatomic, getter=isUsingBytecodeCache, readonly) BOOL usingBytecodeCache;
```

Comment:
I can't do much more here since I don't know Objective C, but there are still a lot of API changes in this pull request. Looks like they might possibly be backwards-compatible and therefore OK? Not sure. It's going to require Apple API review regardless.

Reply:
All backwards compatible.

Reply:
> I can't do much more here since I don't know Objective C, but there are still a lot of API changes in this pull request. Looks like they might possibly be backwards-compatible and therefore OK? Not sure. It's going to require Apple API review regardless.

Do you know who can do this?